AmplifyPlugins/API/AWSAPICategoryPluginIntegrationTests/GraphQL/GraphQLWithLambdaAuthIntegrationTests/README.md

```diff
@@ -17,7 +17,7 @@ The following steps demonstrate how to set up an GraphQL endpoint with AppSync.
 ? Choose a schema template: `Single object with fields (e.g., “Todo” with ID, name, description)`
 ```
 
-3. `amplify push`
+3. If you are using the latest CLI, update cli.json to include `"useExperimentalPipelinedTransformer": false` to ensure that it will use the v1 transformer and then `amplify push`
```

Comment:
- can you expand this to full path? ie. `useExperimentalPipelinedTransformer` -> `features.graphqltransformer.useExperimentalPipelinedTransformer: false`
- and also add `features.graphqltransformer.transformerversion: 1`

Reply:
Do we need to change this: `features.graphqltransformer.transformerversion: 1` it worked for me without changing it.

Reply:
i remember changing it but if `useExperimentalPipelinedTransformer: false` was enough, let's stick with that